packages/client/src/object/convertWireToOsdkObjects.test.ts

```diff
@@ -167,7 +167,7 @@ describe("convertWireToOsdkObjects", () => {
       clientCtx,
       [objectFromWire],
       undefined,
-    )) as Osdk<Employee>[];
+    )) as unknown as Osdk<Employee>[];
```

Comment:
Had to do this because `Osdk<>` type is pretty complex now so it has a hard time casting from `Osdk<ObjectOrInterfaceDefinition>` to `Osdk<myConcreteType>`